templates/types/streaming/express/package.json

```diff
@@ -20,7 +20,7 @@
     "dotenv": "^16.3.1",
     "duck-duck-scrape": "^2.2.5",
     "express": "^4.18.2",
-    "llamaindex": "0.5.8",
+    "llamaindex": "0.5.12",
```

Comment:
TODO: use version that includes https://github.com/run-llama/LlamaIndexTS/pull/1088